tools/run_tests/run_tests_matrix.py

```diff
@@ -411,6 +411,8 @@ def _runs_per_test_type(arg_str):
     extra_args.append('--bq_result_table')
     extra_args.append('%s' % args.bq_result_table)
     extra_args.append('--measure_cpu_costs')
+    extra_args.append('--auto_set_flakes')
```

Comment:
This itself looks good, but it also looks like it's impossible to disable the --auto_set_flakes because of 
python args library wierdness (try --auto_set_flakes False, --auto_set_flakes=false - none of them works).
https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse

So we need to change the way --auto_set_flakes gets registered in run_tests.py and only then fix here.